<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelAdresse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresse">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message de la banque : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>561</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperations">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque :</string>
    </property>
    <widget class="QPushButton" name="pushButtonNumeroCompte">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumeroCompte">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMontant">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMontant">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSolde">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Solde</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRetrait">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrait</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDepot">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dépôt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnvoi">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidgetEtat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
